apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,40 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { CityStore } from '../../data-access/city.store';
+import { CardType } from '../../model/card.model';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { ListItemDirective } from '../../ui/list-item-directive/list-item.directive';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `<app-card
+    [list]="cities()"
+    [type]="cardType"
+    customClass="bg-lightyellow">
+    <ng-template listItem let-rowItem>
+      <app-list-item [id]="rowItem.id" [type]="cardType">{{
+        rowItem.name
+      }}</app-list-item>
+    </ng-template>
+  </app-card>`,
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent, ListItemDirective],
+  styles: [
+    `
+      .bg-lightyellow {
+        background-color: lightyellow;
+      }
+    `,
+  ],
+  encapsulation: ViewEncapsulation.None,
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  constructor(private store: CityStore, private http$: FakeHttpService) {}
```

Comment:
i think thats a typo right here `http$`? 😄

Reply:
$ should only be used for observable as a convention.